### PR TITLE
s390x: workaround broken dasdconfigure

### PR DESCRIPTION
**Pull request by Soulofdestiny**
set of verification runs:
https://openqa.suse.de/tests/overview?build=Soulofdestiny%2Fos-autoinst-distri-opensuse%238738&distri=sle&version=12-SP5

### CODE REVIEW

File: tests/installation/bootloader_s390.pm [hunk 1]
```diff
@@ -262,7 +262,9 @@ sub format_dasd {
     }
 
     # bring DASD down again to test the activation during the installation
-    assert_script_run("dasd_configure 0.0.0150 0");
+    # currently broken due to bsc#1151394
+    return 1 && record_soft_failure('bsc#1151394');
+    # assert_script_run("dasd_configure 0.0.0150 0");
```
Comment (foursixnine) [inline tests/installation/bootloader_s390.pm]: Can you remove this line?

File: tests/installation/bootloader_s390.pm [hunk 2]
```diff
@@ -262,7 +262,9 @@ sub format_dasd {
     }
 
     # bring DASD down again to test the activation during the installation
-    assert_script_run("dasd_configure 0.0.0150 0");
+    # currently broken due to bsc#1151394
+    return 1 && record_soft_failure('bsc#1151394');
```
[the remark this answers is not in the document]
Comment (Soulofdestiny) [reply tests/installation/bootloader_s390.pm]: well - yeah sounds logical, but according to verification runs it works.
anyway, let's be on the safe side here

**Comment by Soulofdestiny** [comment]
@foursixnine @SergioAtSUSE thanks for suggestions, fixed =)

**Comment by Soulofdestiny** [comment]
self-merge to possibly spot issues in GMC Candidate

**Pull request merged**